src/els_parser.erl

```diff
@@ -114,11 +115,8 @@ do_find_spec_points_of_interest(Tree, Acc) ->
 
 -spec points_of_interest(tree()) -> [poi()].
 points_of_interest(Tree) ->
-  lists:flatten(
-    erl_syntax_lib:fold(
-      fun(T, Acc) ->
-          [do_points_of_interest(T)|Acc]
-      end, [], Tree)).
+  FoldFun = fun(T, Acc) -> do_points_of_interest(T) ++ Acc end,
```

Comment:
Do we need the `++` in here?

Reply:
Do you mean at the end as in `parse/1` or at the end of the fold? If it's the former then I would agree, if it's the latter, I think it's much clearer what's happening if we use the `++`.